src/daemon/rpc_command_executor.cpp

```diff
@@ -2284,6 +2294,12 @@ bool t_rpc_command_executor::print_sn(const std::vector<std::string> &args)
       }
     }
 
+    if (include_json)
```

Comment:
I think the convention currently in the codebase is if +json is requested, it only prints out the json representation otherwise print out the data normally.